plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -2,13 +2,332 @@
+		</table>',
+        props: {
+            parameter: {
+                type: Object,
+                default: function() {
+                    return {};
+                }
+            }
+        },
+        computed: {
+            conditions: function() {
+                var conditions = [];
+                var conditionsMap = this.$store.getters["countlyRemoteConfig/parameters/conditionsMap"];
+                var self = this;
+                if (this.parameter.conditions.length > 0) {
+                    this.parameter.conditions.forEach(function(condition) {
+                        var conditionProperties = conditionsMap[condition.condition_id];
+                        var ob = {
+                            color: self.getColor(conditionProperties.condition_color),
+                            percentage: condition.c ? ((condition.c) / self.totalConditions).toFixed(2) : 0,
+                            name: conditionProperties.condition_name,
+                            value: condition.value
+                        };
+                        conditions.push(ob);
+                    });
+                }
+                return conditions;
+            },
+            totalConditions: function() {
+                var total = 0;
+                if (this.parameter.conditions.length > 0) {
+                    this.parameter.conditions.forEach(function(condition) {
+                        total = total + condition.c ? condition.c : 0;
+                    });
+                }
+                return total;
+            },
+            defaultValue: function() {
+                var ob = {
+                    value: this.parameter.default_value,
+                    percentage: this.parameter.c ? ((this.parameter.c) / this.totalConditions).toFixed(2) : 0,
+                };
+                return ob;
+            }
+        },
+        methods: {
+            getColor: function(condition_color) {
+                var arr = COLOR_TAG.filter(function(item) {
+                    return item.value === condition_color;
+                });
+                return arr[0].label;
+            }
+        }
+    });
+    var JsonEditor = countlyVue.views.create({
+        template: CV.T("/remote-config/templates/json-editor.html"),
+        data: function() {
+            return {
+                currentVal: ""
+            };
+        },
+        computed: {
+        },
+        props: {
+            value: String,
+            isOpen: Boolean
+        },
+        watch: {
+            value: {
+                immediate: true,
+                handler: function(newValue) {
+                    this.currentVal = newValue;
+                }
+            },
+        },
+        methods: {
+            handleClose: function() {
+                this.$store.dispatch("countlyRemoteConfig/parameters/showJsonEditor", false);
+                this.$store.dispatch("countlyRemoteConfig/parameters/showJsonEditorForCondition", false);
```

Comment:
`showJsonEditor` and `showJsonEditorForCondition ` can be local states or event props with default values of this component.

Reply:
Reference to [this](https://github.com/Countly/countly-server/pull/2171/files?diff=unified&w=0#r759976037) and [this](https://github.com/Countly/countly-server/pull/2171/files?diff=unified&w=0#r759977398)